3rdparty/jumbojett/openid-connect-php/src/OpenIDConnectClient.php

```diff
@@ -361,8 +364,6 @@ public function authenticate() {
             // Save the id token
             $this->idToken = $token_json->id_token;
 
-            // Save the access token
-            $this->accessToken = $token_json->access_token;
 
```

Comment:
The access token must be saved earlier in the flow, otherwise it is not available later on.